### ISSUE TITLE
Set up redirects to docs.asciidoctor.org

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor.org#818]
The documentation is currently being migrated from asciidoctor.org/docs to docs.asciidoctor.org. Once the migration is complete, and docs.asciidoctor.org is live, the docs folder should be removed from this site and replaced with rewrite rules that redirect the current pages to matching pages in the new docs site.

For the most part, this is just a 1-to-1 mapping. These redirects should be done using [Netlify redirects](https://www.netlify.com/docs/redirects/).

The user manual will need special treatment since there are so many deep links to that page. Therefore, the current page should be replaced with a redirector page that inspects the fragment and issues a client-side (JavaScript) redirect to the most suitable page. (This is necessary since the fragment is not visible to a server-side rewrite engine).

**Comment by mojavelinux** [comment]
This is a duplicate of #981 and has since been resolved.
